packages/logger/tsconfig.json

```diff
@@ -16,22 +16,16 @@
     "moduleResolution": "node",
     "baseUrl": ".",
     "paths": {
-      "*": [
-        "./types/*"
-      ]
+      "*": ["./types/*"]
```

Comment:
Other changes are made to fix errors from `npm run lint` 🔍

Reply:
what error would be thrown without this change?

Reply:
```
$ npx @biomejs/biome check .
./tsconfig.json format ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Formatter would have printed the following content:

    17 17 │       "baseUrl": ".",
    18 18 │       "paths": {
    19    │ - ······"*":·[
    20    │ - ········"./types/*"
    21    │ - ······]
       19 │ + ······"*":·["./types/*"]
    22 20 │       },
    23 21 │       "esModuleInterop": true
    ····· │
    27 25 │       // "resolveJsonModule": true,
    28 26 │     },
    29    │ - ··"include":·[
    30    │ - ····"src/**/*"
    31    │ - ··],
    32    │ - ··"exclude":·[
    33    │ - ····"src/**/*.spec.*"
    34    │ - ··],
       27 │ + ··"include":·["src/**/*"],
       28 │ + ··"exclude":·["src/**/*.spec.*"],
    35 29 │     "jsdoc": {
    36 30 │       "out": "support/jsdoc",


Checked 9 files in 3ms. No fixes applied.
Found 1 error.
check ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ Some errors were emitted while running checks.


```

An expection of an inlined list. What a beautiful error message this is 🤧

My output even shows `diff` with red and green 🚙 💨  TBH I would've kept the separate lines but I agree with the linter 🙏

Reply:
This is even an error that was changed with automagic 🪄  Quite impressive! ✨